src/Pages/HumansAtNasa/HumansAtNasa.tsx

```diff
@@ -1,16 +1,17 @@
-﻿﻿import React from "react";
+import {HumansInSpace} from "../../Components/HumansInSpace/HumansInSpace";
+import React from "react";
 import {HumanAtNasa} from "../../Components/HumanAtNasa/HumanAtNasa";
 import './HumansAtNasa.scss';
 
 export function HumansAtNasa() {
     return (
-        <section className='humansAtNasaBody'>
-            <h1>Humans At Nasa</h1>
+        <div className='humansAtNasaBody'>
```

Comment:
Why did we change it from `section` to `div`?

Reply:
not sure, changed back :)

Reply:
still seems to be a div...

Reply:
changed on other branch